Sources/Orbit/Support/Previews/PreviewDisplayName.swift

```diff
@@ -0,0 +1,23 @@
+import SwiftUI
+
+public extension View {
+
+    /// Sets a name to show in the canvas for a preview, based on a function name.
```

Comment:
```suggestion
    /// Sets a name to show in the canvas for a preview, based on the preview property name.
```

Reply:
Will update. Its just that the internal parameter is called `function` - should I rename it to be `property`, as we expect to use it on properties only?

Reply:
Well, we expect it to be used on properties, since the previews are computed `var`s...

Reply:
Yup, updated